lib/url_launcher.dart

```diff
@@ -114,7 +116,11 @@ Future<bool> launchLink({
 }) async {
   final uri = Uri.tryParse(link);
   if (uri != null && await ul.canLaunchUrl(uri)) {
-    await ul.launchUrl(uri);
+    await ul.launchUrl(uri, mode: 
+        context.read<ConfigStore>().openLinksInExternalApp ?
+        ul.LaunchMode.externalApplication : 
+        ul.LaunchMode.inAppWebView
```

Comment:
The false case should be `platformDefault`. There are cases where we shouldn't try to open in web view even without this setting.